.github/workflows/build-and-run.yml

```
@@ -0,0 +1,109 @@
+name: Build-And-Run Tests
+
+on: [push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_and_test:
+    env:
+      ALL_EXCLUDE: stage_dem|ionosphere|background|pybind.unwrap.phass
```

`ctest -E` uses a regex pattern to identify tests to exclude, so a`.` like this will match any character. If we want to match a literal `.` instead, we should escape the dot with a backslash, like this:

```suggestion
      ALL_EXCLUDE: stage_dem|ionosphere|background|pybind\.unwrap\.phass
```